mmdet/utils/deployment/openvino_backend.py

```diff
@@ -157,27 +157,28 @@ def configure_outputs(self, net, required_outputs):
                 if output in net.outputs:
                     net_outputs_mapping[output] = required_output
                 else:
-                    raise ValueError('Failed to identify output {}'.format(required_output))
+                    raise ValueError('Failed to identify output "{}"'.format(required_output))
         return net_outputs_mapping
 
     def rename_outputs(self, outputs):
         return {self.net_outputs_mapping[k]: v for k, v in outputs.items() if k in self.net_outputs_mapping}
 
-    def forward(self, inputs):
+    def normalize_inputs(self, inputs):
```

Comment:
is it a good name? maybe `remap_inputs` or `fill_inputs` or something? but not normalize

Reply:
Renamed to `unify_inputs`.